pages/linux/spectre-meltdown-checker.md

```diff
@@ -0,0 +1,24 @@
+# spectre-meltdown-checker
+
+> Spectre and Meltdown mitigation detection tool.
+> More information: <https://manned.org/spectre-meltdown-checker.1>.
+
+- Check the currently running kernel for Spectre and Meltdown:
+
+`spectre-meltdown-checker`
+
+- Check the currently running kernel and show explanations for taken actions to mitigate a vulnerability:
+
+`spectre-meltdown-checker --explain`
+
+- Don't use the `/sys` interface even if present:
```

Comment:
How about moving this example down below the next example, i.e. to the 4th position?  the next one seems more essential.

Reply:
Sounds reasonable